packages/devtools-source-map/assets/README.md

```diff
@@ -0,0 +1 @@
+The assets directory is mandatory to run `../bin/publish-assets.js`.
```

Comment:
this file was a dirty workaround to make sure the assets directory was available. Since you have firefox/moz.build here, you can remove it.

And we should simply create the assets/ folder if it doesn't exist in the publish-assets script.